packages/next/src/next-devtools/server/middleware-response.ts

```diff
@@ -6,9 +6,14 @@ export const middlewareResponse = {
     res.statusCode = 204
     res.end('No Content')
   },
-  badRequest(res: ServerResponse) {
+  badRequest(res: ServerResponse, reason?: string) {
```

Comment:
Nit:
```suggestion
  badRequest(res: ServerResponse, reason = 'Bad Request') {
```
To preserve previous behavior for existing call sites.

Reply:
I wanted to change the previous behavior. Having "Bad Request" in the response body and status text seems redundant.